### ISSUE TITLE
Regression: Blazblue: Centralfiction (586140) crashes when selecting specific character (Mai Natsume)

### ISSUE THREAD

**Issue by alasky17** [doitsujin/dxvk#3142]
### Software information
Blazblue: Centralfiction (586140)
Reported by folks here: https://github.com/ValveSoftware/Proton/issues/1031#issuecomment-1355716116

Blazblue: Centralfiction crashes when selecting Mai Natsume.  The easiest way to test this is: 
1) Launch game and get to main menu
2) Under "Practice", select Training Mode
3) Select Mai Natsume as a character - she is in the very top right corner of the selection screen with purple hair

### System information
Reproduced on Steam deck
- GPU: AMD custom GPU 0405 (vangogh, LLVM 13.0.0, DRM 3.45, 5.13.0-valve21,3-1-neptune)
- Driver: Mesa 22.0.2 (git-71648faf15) - on SteamOS Stable
- Wine version: Broken in current Proton Experimental-7.0-20221208 and bleeding edge (experimental-bleeding-edge-7.0-31585-20221220-p1a1b4a-w30ed35-d12901b-ve8d00f).  Working with Proton 7.0-6 on frozen DXVK branch
- DXVK version: HEAD at 12901b52f19ea1d3eece450a009f4602b5f2adc3

### Apitrace file(s)
Instead of getting this, I bisected the regression.  It appeared to bisect cleanly and blamed: 
commit 6ca6554452543cca14daf017437c5b7b3e848909
Author: Robin Kertels <robin.kertels@gmail.com>
Date:   Wed Mar 16 02:28:22 2022 +0100

    [d3d9] Use memory mapped files for textures

Please let me know if apitrace would still be useful to try to obtain.

### Log files
- d3d9.log:[BBCF_d3d9.log](https://github.com/doitsujin/dxvk/files/10272152/BBCF_d3d9.log)

**Comment by alasky17** [comment]
I got an apitrace of crash on experimental and working dxvk version (7.0-6).  Thank you for the help Blisto91! The files are way too big to upload, so I uploaded them here with unlock code "apitrace" : https://www.codeweavers.com/xfer/alasky/blazblue-traces/

**Comment by K0bin** [comment]
Unfortunately both apitraces replay just fine. I don't have access to the game either.

**Comment by PhantomX** [comment]
`d3d9.textureMemory = 0` fixed for me.

**Comment by Blisto91** [comment]
Yes it's a issue with the new memory unmapping feature introduced in 2.0. The above option disables it when set to 0.

**Comment by K0bin** [comment]
This appears to be an off-by-one bug in the game. If I pad the allocation for the texture by literally just 1 byte, it works.

**Comment by Isaiahmag** [comment]
This now happens again in Proton 8.0-1 again.

**Comment by K0bin** [comment]
That's most likely an unrelated issue then. Please report that over on the Proton issue tracker.

**Comment by Isaiahmag** [comment]
> That's most likely an unrelated issue then. Please report that over on the Proton issue tracker.

Oh sorry I thought I was lol
